.env.example

```diff
@@ -1,11 +1 @@
-CENTRE_DETAILS_FILE_PATH=crawler/config/centre_details.json
-MONGO_DB=crawlerDevelopmentDB
-MONGO_HOST=127.0.0.1
-MONGO_PORT=27017
-SFTP_HOST=<sftp_host>
-SFTP_PASSWORD=<sftp_password>
```

Comment:
was looking into these, and it seems SFTP_PASSWORD is obsolete, but mentioned in test.py - so maybe remove it from there too?

Reply:
Thanks, have removed it from test.py too.